Aandachtspunten.md

```diff
@@ -1,9 +1,28 @@
 ## Aandachtspunten
 
+### Inhoud
+
 - Alle randvoorwaarden zijn voldaan.
 
-- Er wordt gebruik gemaakt van Dependency Injection via de constructor OF via @Autowired
+- Er wordt gebruik gemaakt van Dependency Injection via de constructor (OF via @Autowired, maar dat heeft niet de voorkeur
+
+- Er zijn Entities gemaakt voor Television, CiModule, WallBracket en RemoteController die inhoudelijk overeenkomen met de opdrachtbeschrijving
 
-- De service-laag verstuurd geen Television objecten naar de controller, enkel TelevisionDto’s
+- Er is een OneToOne relatie tussen Television en RemoteController. De relatie staat maar aan 1 kant, als de relatie aan 2 kanten staat (dus in beide klassen een @OneToOne), dan heeft een van beide kanten ook een "mappedBy"
+- Er is een ManyToOne relatie tussen CiModule en Television. In de Television klasse staat de @ManyToOne annotatie met optioneel de fetchtype en cascade en ook optioneel een @JoinColumn. In de CiModule klasse staat optioneel een @OneToMany annotatie. Als deze er staat, moet  deze een "mappedBy" hebben.
+- Er is een @ManyToMany relatie tussen Television en WallBracket. Optioneel staat deze annotatie aan beide kanten, als dat zo is, dan heeft een van beide kanten verplicht een "mappedBy". Optioneel kan de andere kan een @JoinTable annotatie hebben (om de tabelnaam en kolom namen van de join tabel aan te passen)
+
+- De service-laag verstuurd geen Entity objecten naar de controller, enkel TelevisionDto’s
```

Comment:
Het is goed om te zien dat je DTO's gebruikt om gegevens te transporteren tussen je lagen. Dit helpt om losjes gekoppelde architecturen te behouden.